docker-compose.edgex.yml

```diff
@@ -0,0 +1,197 @@
+# Copyright © 2020-2022 Intel Corporation. All rights reserved.
```

Comment:
These compose file updates should be in a separate PR.

Reply:
> These compose file updates should be in a separate PR.

it was in separated PR and already merged into Edgex2.2 branch, this happens to be here because I did a rebase of Edgex2.2

Reply:
A rebase should exclude it.